grass7/raster/r.centroids/r.centroids.html

```diff
@@ -0,0 +1,50 @@
+<h2>DESCRIPTION</h2>
+
+<p>
+<em>r.centroids</em> is a wrapper function for <a href="https://grass.osgeo.org/grass-stable/manuals/r.volume.html">r.volume</a>
+that computes the center point of raster clumps of data (neighboring, same-valued pixels). The centroids will always fall within
+the clump so they may not be the true, mathematical centroid. The centroids will always fall at the center of a pixel.
+<p>
+
+<p>Options and flags:<p>
```

Comment:
Remove the heading/text. Leave only `<p>` for the next sentence/paragraph. (See also comments in the conversion from the initial review.)

Reply:
Saw the comments from the initial review and I think it makes sense now. Third time's the charm.